pnpm-lock.yaml

```diff
@@ -64,7 +64,7 @@ importers:
         version: 18.19.119
       typescript:
         specifier: ^5.3.3
-        version: 5.6.3
+        version: 5.8.3
```

Comment:
We may have accidentally upgraded TypeScript for Kit here and that affects our generated types. This seems to happen a lot. Should we pin the TypeScript version in Kit for now?